lib/mongodb.js

```diff
@@ -816,18 +843,83 @@ MongoDB.prototype.all = function all(model, filter, options, callback) {
     query = self.buildWhere(model, filter.where);
   }
   var fields = filter.fields;
+  var relations = this._models[model].settings.relations;
```

Comment:
I don't know, the code is not working on using mongodb.test.js only.
When testing the api via http request I get the models relations.